trieste/bayesian_optimizer.py

```diff
@@ -910,15 +911,22 @@ def _write_summary_step(
                     logging.scalar(f"query_points/[{i}]", float(query_points[0, i]))
                 else:
                     logging.histogram(f"query_points/[{i}]", query_points[:, i])
+            logging.histogram("query_points/euclidean_distances", pdist(query_points))
 
         if pd and sns and logging.include_summary("query_points/_pairplot"):
             columns = [f"x{i}" for i in range(tf.shape(query_points)[1])]
-            query_new_df = pd.DataFrame(query_points, columns=columns).applymap(float)
+            qp_preds = query_points
```

Comment:
What exactly is this doing? Is it changing the pairplots to show the predictions at the query points rather than the query points themselves? Have you run this past Victor (who originally demanded this plot)?

Reply:
That's right, adds predictions at qp's. This is uncontraversial change, qp's are still there and this adds useful information.